packages/components/src/navigation/README.md

```diff
@@ -1,26 +1,74 @@
 # Navigation
 
-Navigation is a component which renders a component which renders a heirarchy of menu items.
+Render a heirarchy of menu items into a waterfall style navigation.
 
 ## Usage
 
 ```jsx
-import { MenuItem } from '@wordpress/components';
-import { withState } from '@wordpress/compose';
+import Navigation from '@wordpress/components';
```

Comment:
I don't think we can import from the default and the names may include "experimental" for a while; not sure if worth noting here:

```suggestion
import {
	__experimentalNavigation as Navigation,
	__experimentalNavigationMenu as NavigationMenu,
	__experimentalNavigationMenuItem as NavigationMenuItem,
} from '@wordpress/components';
```

Reply:
Probably a good idea. There are instances of each present in the codebase.

https://github.com/WordPress/gutenberg/blob/ebd5d9605f0aa57088f74d0ecaccf93017e36d5e/packages/components/src/number-control/README.md#L8

https://github.com/WordPress/gutenberg/blob/ebd5d9605f0aa57088f74d0ecaccf93017e36d5e/packages/components/src/text/README.md#L8